variables.tf

```diff
@@ -86,6 +58,23 @@ If it is set to false, then no telemetry will be collected.
 DESCRIPTION
 }
 
+variable "identity_ids" {
+  type        = list(string)
+  default     = null
+  description = "(Optional) Specifies a list of User Assigned Managed Identity IDs to be assigned to this Kubernetes Cluster."
+}
+
+variable "identity_type" {
+  type        = string
+  default     = "SystemAssigned"
```

Comment:
Did we have `UserAssigned` in our design document ?

Reply:
i've updated this, let me know if you can run this and get the user assigned managed identity